test/e2e/serial/tests/configuration.go

```diff
@@ -48,7 +48,7 @@ import (
 	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 )
 
-var _ = Describe("[serial][disruptive][slow] configuration management", func() {
+var _ = Describe("[serial][disruptive][slow] numaresources configuration management", func() {
```

Comment:
If this should be used as a label maybe it's better to set it as a label format, i.e. ```[numaresources]```

Reply:
this is what I would like to do myself, but it seems to be going against the standard practice in cnf-tests. For the time being, I'd rather try to blend in their ecosystem.